web/js/scripts.js

```diff
@@ -199,6 +199,18 @@ var grouphub = (function ($) {
             updatePanelsCookie($group.attr('id'), checked);
         });
 
+        $groupContainer.on('click', '.toggle_extra_group_info', function () {
+            var $extraGroupInfo = $(this).parents('li').find('.extra-group-info');
+            var $span = $(this).find('span');
+
+            $extraGroupInfo.toggleClass('hidden');
+
+            $span.toggleClass('fa-angle-double-down');
+            $span.toggleClass('fa-angle-double-up');
```

Comment:
Not sure if possible (and if it is more readable), but these two lines could perhaps be shortened to `$span.toggleClass('fa-angle-double-down fa-angle-double-up');`